easybuild/easyconfigs/a/ASAP/ASAP-2.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,68 @@
+easyblock = 'CMakeMake'
+
+name = 'ASAP'
+version = '2.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = "https://computationalpathologygroup.github.io/ASAP/"
+description = """ASAP focuses on fast and fluid image viewing with an easy-to-use interface 
+for making annotations. It consists of two main components: an IO library for reading and writing 
+multi-resolution images and a viewer component for visualizing such images."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://github.com/computationalpathologygroup/ASAP/archive']
+sources = ['%(name)s-%(version)s.tar.gz']
+patches = [
+    'ASAP-2.0_libjpeg.patch',
+    'ASAP-2.0_use_pugixml_shared.patch',
+]
+checksums = [
+    'e5d8b0aff6f86a3c7319062ca1f9e35b867da1c80667a80b55ee2a876f897202',  # ASAP-2.0.tar.gz
+    'f593599b5af90c8256ffeae910757d02f4de04a49de26d5123c209a405ba6a55',  # ASAP-2.0_libjpeg.patch
+    '785a7716cbbf7fd4f6dacf1b74fe66384be76d31d54a02918ce9c07989ac4c81',  # ASAP-2.0_use_pugixml_shared.patch
```

Comment:
The CI is complaining that this checksum is not correct, probably because of the typo that was fixed yesterday. I see that you made a commit with a message that it should fix the checksum, but it looks like it didn't actually change a checksum?

Reply:
Silly: I used `--inject-checksums`, but forgot to get the updated patch file from PR. Another downside to just committing suggested changes: your local copy goes out of date =)